config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -889,10 +895,30 @@ def generate_misc():
                    ],
                    focus_regex=r'\[Conformance\]|\[NodeConformance\]',
                    skip_regex=r'\[FOOBAR\]', # leaving it empty will allow kops to add extra skips
+                   test_args="-num-nodes=3 --master-os-distro=gci --node-os-distro=gci",
                    test_timeout_minutes=200,
+                   test_parallelism=1, # serial tests
                    extra_dashboards=["sig-cluster-lifecycle-kubeup-to-kops"],
                    runs_per_day=6),
 
+        build_test(name_override="ci-kubernetes-e2e-cos-gce-conformance-concurrrency-canary",
+                   cloud="gce",
+                   distro="cos105",
+                   networking="kubenet",
+                   k8s_version="ci",
+                   kops_version="https://storage.googleapis.com/kops-ci/bin/latest-ci.txt",
+                   kops_channel="alpha",
+                   build_cluster="k8s-infra-prow-build",
+                   extra_flags=[
+                       "--image=cos-cloud/cos-105-17412-156-49",
+                   ],
+                   focus_regex=r'\[Conformance\]',
+                   skip_regex=r'\[FOOBAR\]', # leaving it empty will allow kops to add extra skips
+                   test_timeout_minutes=100,
+                   test_args="-num-nodes=3 --master-os-distro=gci --node-os-distro=gci --ginkgo.flake-attempts=3", # pylint: disable=line-too-long
```

Comment:
That is not valid for our CI, we removed flake attempt some time ago because it hides bugs

Reply:
please remove flake-attempts, there has to be zero tolerance to flakes, I sent an email and updated our docs with that ... the rest lgtm

Reply:
It is an entirely new job, I have a proper conformance job running with no flake-attempts allowed and parallel of 1.

Reply:
yeah, but why should we allow retries on flakes?

Reply:
Looking at the board, it seems that a small set of conformance tests struggle with high parallelism. 

Conformance test in serial takes 2h40m
Conformance test in parallel(30) takes around 40m, but at the cost of a few flakes from the tests not being robust enough.


Is this worth investigating or not?

Reply:
For example, presubmits can run "quick conformance" tests and the existing "legit" conformance test